Automattic/wp-calypso
# Update the Charts on the Google My Business Stats Page

## Specs

This PR will replace all existing google-my-business stats charts with three pie charts: 
1. search results ( direct and discovery ) 
2. views ( maps and search ) 
3.  customer actions ( visit, call, or request directions )

Each of these charts should be populated with fake data. A user should be able to independently switch the interval of the data from week, month, and quarter on each chart.

## Testing

1. Connect a Google My Business eligible site
2. Navigate to the stats page and the new Google My Business pane
3. Verify that the page is looks very similar to the following screenshot.
<img width="732" alt="screen shot 2018-04-13 at 7 44 10 pm" src="https://user-images.githubusercontent.com/2810519/38763542-1f03a8c2-3f53-11e8-9711-f61abe474db8.png">

4. Change the interval on each chart. Verify that the values of the series and the total changes as the intervals change

## Review
